docs/_specs/application/application.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+---
+
+# Application-scoped Dispatch
+
+## Introduction
+
+Within Dispatch, we believe the concept of “Application” is a very important aspect. In a non-trivial deployment of dispatch, Applications group and isolate resources of different function unit, provide the ability of searching and filtering resources; they also facilitate permission management and access controls.
+
+In addition to those basic functionalities, the concept of Application also provides a step forwards to advanced features of cloud native apps, e.g. lifecycle management (i.e. creation/delete, import and export Applications, scaling up and down based on demand/usage);, function execution workflow management, visualized workflow GUI; application level monitoring and analyzing; function/application external dependencies management.
+
+In this design documents, we outline several design concerns and implementation decisions with regards to the concept of a Serverless Application.
+
+Note: there is another grouping concept “Organization” within the realm of Dispatch. As its detail definition hasn't been finalized, and it is out of the scope of this articles; we decide to only discuss “Application” under the assumption of a single tenant dispatch deployment, in which the system scope and the organization scope are equivalent, and the system admin and the organization admin are equivalent.
```

Comment:
You could probably just remove this note.

Reply:
sure